getting-started/scaling-out.md

```diff
@@ -0,0 +1,144 @@
+*distributed hypertables*.
+Distributed hypertables are similar to normal hypertables, but they
+add an additional layer of hypertable partitioning by distributing chunks
+across *data nodes*.
+
+In the multi-node topology ([architecture][]), all nodes are TimescaleDB instances.
+The data are distributed and stored in TimescaleDB instances, called data nodes.
+The distributed data are accessed through distributed hypertable
+on another TimescaleDB instance, called the *access node*.
+The access node is the entry point for any access to data in the cluster.
+
+## Working with Data Nodes
+
+Data nodes act as the containers for the hypertable chunks and are
+necessary to create distributed hypertables. Data nodes are
+added to the current database using [`add_data_node`][add_data_node]
+and removed using [`delete_data_node`][delete_data_node].
+
+Note that:
+
+* Data nodes are added as objects locally and just connect to an
+  existing database server.
+
+* You should already have a running database server on some host.
+
+When creating the data node, you should:
+
+* Provide a (local) name to use when referring to the data node from
+  this database.
+
+* Provide the host where the hypertable partition for the distributed
+  hypertable should be stored.
+
+* Ensure that the user used for connecting to the data node has
+  `CREATEDB` permissions or be a superuser. This is necessary since
+  `create_distributed_hypertable` expect to be able to create a
+  database on the remote data node.
+
+```sql
+SELECT add_data_node('node1', host => 'dn1.example.com');
+SELECT add_data_node('node2', host => 'dn2.example.com');
+```
+
+Deleting a data node is done in a similar manner.
+
+```sql
+SELECT delete_data_node('node1');
+```
+>:TIP: Note that a data node cannot be deleted if it contains data for a
+hypertable.
+
+### Information Schema for Data Nodes
+
+The data nodes that have been added to the database can be found by
+querying the
+[`timescaledb_information.data_node`][timescaledb_information-data_node] view.
+
+## Working with Distributed Hypertables
+
+As previously mentioned, distributed hypertables are similar to normal
+hypertable with the difference that distributed hypertables will
+partition the data and store it on remote data nodes. This means that
+to create a distributed hypertable, you need to have added at least one data node
+before starting.
+
+### Creating Distributed Hypertables
+
+As with normal hypertables, start by creating a regular
+table to hold the data, then transform it into a distributed
+hypertable. In contrast to when creating a normal hypertable, we use
+the function
+[`create_distributed_hypertable`][create_distributed_hypertable].
+
+```sql
+CREATE TABLE conditions (
+  time        TIMESTAMPTZ       NOT NULL,
+  location    TEXT              NOT NULL,
+  temperature DOUBLE PRECISION  NULL,
+  humidity    DOUBLE PRECISION  NULL
+);
+
+SELECT create_distributed_hypertable('conditions', 'time');
```

Comment:
I notice a couple of resolved conversations (without response) about adding a space dimensions here, but I still don't see the statement changed.  Please either make the change or comment why it's not being done.

Reply:
It was resolved, but disappeared during merge hell.

Reply:
Done again.